code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -325,28 +325,6 @@
 	sharp = TRUE
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut", "honked")
 
-/obj/item/nullrod/whip
-	name = "holy whip"
-	desc = "A whip, blessed with the power to banish evil shadowy creatures. What a terrible night to be in spess."
-	icon_state = "chain"
-	item_state = "chain"
-	slot_flags = SLOT_BELT
-	attack_verb = list("whipped", "lashed")
-	hitsound = 'sound/weapons/slash.ogg'
-
-/obj/item/nullrod/whip/New()
-	..()
-	desc = "What a terrible night to be on the [station_name()]."
-
-/obj/item/nullrod/whip/afterattack(atom/movable/AM, mob/user, proximity)
-	if(!proximity)
-		return
-	if(ishuman(AM))
-		var/mob/living/carbon/human/H = AM
-		if(is_shadow(H))
-			var/phrase = pick("Die monster! You don't belong in this world!!!", "You steal men's souls and make them your slaves!!!", "Your words are as empty as your soul!!!", "Mankind ill needs a savior such as you!!!")
-			user.say("[phrase]")
-			H.adjustBruteLoss(12) //Bonus damage
```

Comment:
This is the one thing I need to address. This nullrod is basically useless now since slings wont be here, and the description wouldn't even make sense now.

Open to suggestions.